felles/src/main/kotlin/no/nav/familie/kontrakter/felles/Ressurs.kt

```diff
@@ -42,7 +42,7 @@ data class Ressurs<T>(
 
         private fun Throwable.textValue(): String {
             val sw = StringWriter()
-            this.printStackTrace(PrintWriter(sw!!))
+            this.printStackTrace(PrintWriter(sw))
```

Comment:
Kan du legge til noe logikk for at feilmeldingen også faktisk printes ut? Nå tror jeg ikke det skjer. Gjerne til securelogger så vi sikrer at ikke noe sensitive data lekker ut.